src/components/interface/Button/Button.js

```diff
@@ -51,9 +56,6 @@ Button.propTypes = {
   onClick: PropTypes.func,
   iconPosition: PropTypes.oneOf(['left', 'right']),
   title: PropTypes.string.isRequired,
-  /**
-   * One of: string, object
-   */
   className: CLASS_NAME_TYPE,
```

Comment:
/**
   * One of: string, object
   */

Reply:
Du coup si c'est pour rajouter ca a chaque fois ca ne sert a rien d'avoir une contante. Si on change la constante, on doit repasser partout pour changer la doc... A ce compte la pourquoi ne pas laisser les propTypes directement dans le composant ? Je trouve ca mieux d'ailleurs.

Reply:
J'avoue. Mais en terme de lecture de code c'est plus clair. Mais priorité au styleguide donc on peut retirer `CLASS_NAME_TYPE`